tests/testthat/_snaps/flow.md

```diff
@@ -3,32 +3,32 @@
     Code
       st_cuts(g_path, source = "a", target = NULL)
     Condition
-      Error in `st_cuts()`:
-      ! No vertex was specified
+      Error in `check_vertex()`:
+      ! Can't find vertex corresponding to `target`.
 
 ---
 
     Code
       st_cuts(g_path, source = NULL, target = "a")
     Condition
-      Error in `st_cuts()`:
-      ! No vertex was specified
+      Error in `check_vertex()`:
+      ! Can't find vertex corresponding to `source`.
```

Comment:
the big improvement is getting the argument name in the error message. Rather useful when there are two arguments being checked in the same away.